automation/build-bin.ts

```diff
@@ -0,0 +1,34 @@
+import * as path from 'path';
+import * as fs from 'fs-extra';
+import * as filehound from 'filehound';
+import { exec as execPkg } from 'pkg';
+
+const ROOT = path.join(__dirname, '..');
+
+console.log('Building package...\n');
+
+execPkg([
+	'--target', 'host',
```

Comment:
Not sure if this is something you want to consider, but if the host system is GNU/Linux and has a pretty recent glibc version, then the CLI will only work on very modern GNU/Linux distributions (usually breaking Debian and some other modern + conservative ones).

You can use a Docker container with Ubuntu 12.04 or Debian Wheezy to ensure compatibility, as we do for the Etcher CLI.

Reply:
I'm not going to worry about this for this initial experimental release, but I've opened an issue to test and investigate this further later: https://github.com/resin-io/resin-cli/issues/738